src/pages/landing-page/Categories.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import styles from "./LandingPage.module.css";
+
+export default function () {
```

Comment:
I think you forgot to add the name for the component or am I missing something?

Reply:
If you are exporting component by default, you need not specify any name. Webpack will take the name of the file to be name of the Component.

Reply:
Oh! cool.